admin_guide/service_accounts.adoc

```diff
@@ -64,8 +64,29 @@ To allow all service accounts in the *managers* project to edit resources in the
 ----
 $ oc policy add-role-to-group edit system:serviceaccounts:managers -n top-secret
 ----
-// end::sa-user-names-and-groups[]
 
+[[dedicated-admin-granting-service-accounts-access]]
+ifdef::openshift-dedicated[]
+== Granting Service Accounts Access to Dedicated-Admin Roles
+
+As an {product-title} administrator, you can use service accounts to perform any
+actions that require {product-title} *admin* roles.
+
+The *dedicated-admin* service creates the
+xref:../admin_guide/index.adoc#admin-guide-index-dedicated-admin-role[*dedicated-admins
```

Comment:
Is the `*` meant to be for just "dedicated-admins" or also the word "group"?

Reply:
@adellape Good catch 👁‍🗨 Fixed.